src/example/cxx_grid_example.C

```diff
@@ -281,6 +281,12 @@ int check_if_grackle_mutates_ghost_zone(int primordial_chemistry,
   grackle_data->UVbackground = 1;           // UV background on
   grackle_data->grackle_data_file = "../../input/CloudyData_UVB=HM2012.h5"; // data file
 
+  // Third, check the consistency
+  if (gr_check_consistency() == 0) {
+    fprintf(stderr, "Error in gr_check_consistency.\n");
+    return EXIT_FAILURE;
+  }
+
   // Finally, initialize the chemistry object.
   if (initialize_chemistry_data(&my_units) == 0) {
     fprintf(stderr, "Error in initialize_chemistry_data.\n");
```

Comment:
Just a heads up: You are going to see a merge conflict for this file.

_Context_: While this file is called `cxx_grid_example.C` it isn't really an "example." In reality, it is a test of some basic functionality that is not wrapped by `pygrackle`. For a long time, the pytest test-suite was the only way to run automated tests and thus, it made sense to treat this like a code-example. Recently, we added the ability to run tests directly against the core-library and PR #254 (merged within the last 2 hours) relocated the logic from this file to the core-library test-suite. 

_Just in case you aren't comfortable with resolving merge-conflicts_: the easiest way to avoid the conflict is to revert the changes to this file before merging in more recent commits.

Reply:
Thanks for the heads-up. I reverted the changes in this file.